src/sinks/datadog/traces/apm_stats/aggregation.rs

```diff
@@ -124,7 +124,9 @@ impl Aggregator {
     pub fn new(default_api_key: Arc<str>) -> Self {
         Self {
             buckets: BTreeMap::new(),
-            oldest_timestamp: align_timestamp(Utc::now().timestamp_nanos() as u64),
+            oldest_timestamp: align_timestamp(
+                Utc::now().timestamp_nanos_opt().expect("Invalid timestamp") as u64,
```

Comment:
I see this is only `None` if outside the bounds of an `i64`. Hopefully we fix this before 2262 or no one has timestamps before 1677 🙏 

The `expect` message here didn't really make the conditions under which it would panic clear to me. In this case, it will panic if the date is outside of the "valid range". Maybe we could update it to `.expect("timestamp out of range"`)?

Reply:
Works for me.